author/editors-guide/edition-tips.adoc

```diff
@@ -0,0 +1,219 @@
+---
+layout: author-docs
+title: Metanorma edition tips
+---
+
+The following tips complement the information exposed in this section: link:https://www.metanorma.com/author/ref/asciidoc-tips/[Metanorma AsciiDoc tips]
+
+== Always use lowercase image filenames
+
+If you:
+
+. have a file named `image.PNG` and
+. use this line: `image::image.png[]` to include it,
+
+it will work on macOS and Windows, but will fail on Linux!
+
+Linux uses case-sensitive file systems, i.e. `image.PNG` and `image.png` are different file names
+and Metanorma will fail to find the specified `image.png`.
+
+If you are on Windows, and have a bunch of filenames ending in .PNG, there is a straightforward way to change them all at once:
+
+1. Go to the folder where the images are located.
+2. Place the cursor in the bar address, write "cmd" and press ENTER. The command line will open on that folder location.
+3. In the command line, execute the command `ren *.PNG *.png`, and that's it!
+
+== AsciiMath vs. LaTeX math
+
+Math expressions can be written in AsciiMath or LaTeX math format, which is enabled by `:stem:` document attribute. Default option can be changed from AsciiMath to LaTeX math by using `:stem: latexmath`.
+
+It is recommended to use AsciiMath whenever it is possible, especially since the most of the math expressions can be represented this way. However, some special characters require the help of LaTeX math. Examples of such characters are `\backepsilon` (∍) and `\mathbb{E}` (𝔼). 
+
+In case you are not sure whether the written expression is correct and whether it would render well in either AsciiMath or LaTeX math, you can use https://www.mathjax.org/#demo[MathJax Demo] tool to check the output.
+
+== Math arrays
+
+Do not use the `eqnarray` environment for arrays since it is not supported by `latexmlmath`.
+
+If you want to know if an equation will pass, try this:
+
+`math.lxml`
+[source,latex]
+----
+V = \frac{1}{2} \: {\bf u}^t \:
+            \int_{surface} \: {B'}^t \: D' \: B' \: ds
+               \; {\bf u}
+----
+
+Run:
+
+[source,sh]
+----
+cat math.lxml | latexmlmath -
```

Comment:
Indeed we need to implement this:
* https://github.com/plurimath/plurimath/issues/204

Reply:
I've commented out the related content until we have a consolidated CLI interface.